jsonschema/jsonschema.go

```diff
@@ -0,0 +1,14 @@
+package jsonschema
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"github.com/invopop/jsonschema"
+)
+
+// Generate returns a formatted JSON schema for the parameter.
```

Comment:
```suggestion
// Generate returns a formatted JSON schema for the input struct, according to the tags
// defined by https://github.com/invopop/jsonschema
```

Or something like that maybe? Because the tags you need to add needs to be documented somewhere